runtime/sema/authaccount_type.go

```diff
@@ -0,0 +1,575 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sema
+
+import (
+	"github.com/onflow/cadence/runtime/ast"
+	"github.com/onflow/cadence/runtime/common"
+)
+
+// AuthAccountType represents the authorized access to an account.
+// Access to an AuthAccount means having full access to its storage, public keys, and code.
+// Only signed transactions can get the AuthAccount for an account.
+//
+var AuthAccountType = &NominalType{
+	Name:                 "AuthAccount",
+	QualifiedName:        "AuthAccount",
+	TypeID:               "AuthAccount",
+	IsInvalid:            false,
+	IsResource:           false,
+	Storable:             false,
+	Equatable:            false,
+	ExternallyReturnable: false,
```

Comment:
Probably make `IsContainerType : true` as well?

Reply:
Removed the unused field in 67f13e7d1723c914c3e15a13fec63ad4e1e5d44a